escrow/lib/go/test/templates.go

```diff
@@ -0,0 +1,277 @@
+package test
```

Comment:
Wanted to flag https://github.com/dapperlabs/pinnacle-smart-contracts/blob/main/embed.go
a possible pattern to follow for getting a reference to transactions/contracts/scripts with imports properly mapped for each environment.
It uses an import replacer out of "github.com/onflow/flow-cli/pkg/flowkit/project" -> and also provides transactions suitable for use in testnet/mainnet if you define addresses appropriately in flow.json